apps/app/src/app/pages/(components)/components/(accordion)/accordion.preview.ts

```diff
@@ -72,7 +72,7 @@ import { HlmIconDirective } from '@spartan-ng/helm/icon';
 		</div>
 	`,
 })
-export class AccordionPreviewComponent {}
+export class AccordionPreview {}
 
```

Comment:
I think Line 80 needs a renaming `HlmAccordionContentComponent` -> `HlmAccordionContent`
And all the *.template file.

Similar changes needed in multiple *.preview.ts files (I just did a global search for "Component"/"Directive"). I mentioned some of them here that have a slight documentation change needed.
- apps/app/src/app/pages/(components)/components/(breadcrumb)/breadcrumb.page.ts Line 118
- apps/app/src/app/pages/(components)/components/(breadcrumb)/breadcrumb.preview.ts Line 56, 71, 76

Reply:
Try this regex: `(Component|Directive|Pipe),$`

Reply:
<img width="361" height="253" alt="image" src="https://github.com/user-attachments/assets/4f222f76-7efa-4700-88dc-97e6d3da2691" />